libs/core/src/components/pds-button/pds-button.scss

```diff
@@ -37,18 +44,23 @@
 .pds-button {
   --pds-loader-color: var(--color-text-default);
   align-items: center;
-  background-color: var(--color-background-default);
-  border: var(--pine-border);
+  background-color: var(--pds-button-background);
```

Comment:
I think this might break the background colors applied to all the buttons. You may also need to update the variable naming in the variant styles as well.